dashboard/app/controllers/test_controller.rb

```diff
@@ -397,4 +397,17 @@ def create_user
     sign_in user
     head :ok
   end
+
+  # Accepts a parental request that was submitted by the current user
+  def accept_parental_request
+    permission_request = ParentalPermissionRequest.find_by(user: current_user)
+    Services::ChildAccount.grant_permission_request!(permission_request)
+    head :ok
+  end
+
+  # Ages an account via its created_at property
+  def make_account_older
```

Comment:
Yep! since you added a way to allow `:created_at`, it isn't needed. I originally did what you did, actually, and then thought I shouldn't permit extra things in that weird `users.json` route... then you did the BEST thing and got rid of it haha. I'll remove it.